DockerizeJenkinsfile

```diff
@@ -80,6 +79,44 @@ pipeline {
                             exit $LAST
                         fi
                     popd
+                '''
+            }
+            post {
+                always {
+                    junit testResults: 'uaa/uaa/build/test-results/**/*.xml', allowEmptyResults: true
+                    publishHTML target: [
+                        allowMissing: true,
+                        alwaysLinkToLastBuild: true,
+                        keepAll: true,
+                        reportDir: 'uaa/uaa/build/reports/tests/acceptanceTest',
+                        reportFiles: 'index.html',
+                        reportName: 'Acceptance Test Results'
+                    ]
+                }
+                success {
+                    echo "Acceptance test stage success"
+                }
+                failure {
+                    echo "Acceptance test stage failed"
+                }
+            }
+        }
+        stage('Push Docker Image') {
+            environment {
+                DOCKER_USER = credentials('dtr-docker-repo-pusher')
+            }
+            // Publish the UAA docker image only if we are on release branch
+            when {
+                branch pattern: 'release_\\d+\\.\\d+\\.\\d+_dockerized', comparator: 'REGEXP'
```

Comment:
Remove "_dockerized"; it was added temporarily to publish UAA docker image from release_4.30.0_dockerized branch.

Reply:
Removed